### PR TITLE
Update configuration splash pages

### PR DESCRIPTION
**Pull request by aidanheerdegen**
# ACCESS Community Hive 

Thank you for submitting a pull request to the ACCESS Hive Project. 

## Description

Please include a brief summary of the change and list the issues that are fixed.
Please also include relevant motivation and context.

You can link issues by using a supported keyword in the pull request's description or in a commit message:

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New link / content

## Checklist:

- [x] The new content is accessible and located in the appropriate section.
- [x] My changes do not break navigation and do not generate new warnings
- [x] I have checked that the links are valid and point to the intended content.
- [x] I have checked my code/text and corrected any misspellings

Please tag the **reviewers team** in a comment below by prepending an `@` in front of `ACCESS-Hive/reviewers` when ready.

### CODE REVIEW

**Comment by aidanheerdegen** [comment]
Hey @ACCESS-Hive/reviewers I'm ready for a review

File: docs/configurations/access-cm.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+ACCESS-CM stands for ACCESS **C**oupled **M**odel. This means it is a fully-coupled climate model.
+
+ACCESS-NRI will release an ACCESS-CM model configuration. The first release of ACCESS-CM will be derived from the [CSIRO ACCESS-CM2 configuration](#access-cm2) and will include [atmosphere], [land], [ocean] and [sea ice] components.
+
+## [ACCESS-CM2] {{ recom }}
+
+[**Citation**][ACCESS-CM2-cite] |
+[**Tutorial**][ACCESS-CM2-tute]
+
+ACCESS-CM2 is one of Australia’s contributions to the World Climate Research Programme’s Coupled Model Intercomparison Project Phase 6 (CMIP6). The component models are: UM10.6 GA7.1 for the atmosphere, CABLE2.5 for the land surface, MOM5 for the ocean, and CICE5.1.2 for the sea ice. Compared to previous model versions ACCESS-CM2 shows better global hydrological balance, more realistic ocean water properties (in terms of spatial distribution) and meridional overturning circulation in the Southern Ocean but a poorer simulation of the Antarctic sea ice and a larger energy imbalance at the top of atmosphere. This energy imbalance reflects a noticeable warming trend of the global ocean over the spin-up period.
```
Comment (rbeucher) [inline docs/configurations/access-cm.md:15]: Should we add link to the model components?

File: docs/configurations/access-esm.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+ACCESS-NRI will release an ACCESS-ESM model configuration. The first release of ACCESS-ESM will be derived from the [CSIRO ACCESS-ESM1.5 configuration](#access-esm15) and will include [atmosphere], [land] and [land biogeochemistry], [ocean] and [ocean biogeochemistry], and [sea ice] components.
+
+## [ACCESS-ESM1.5] {{ recom }}
+
+[**Citation**][ACCESS-ESM1.5-cite] |
+[**Tutorial**][ACCESS-ESM1.5-tute]
+
+ACCESS-ESM1.5 is a fully-coupled climate model with land and ocean carbon cycle components. ACCESS-ESM1.5 has mainly been developed to enable Australia to participate in the Coupled Model Intercomparison Project Phase 6 (CMIP6) with an ESM version. An assessment of the climate response to CO2 forcing indicates that ACCESS-ESM1.5 has an equilibrium climate sensitivity of 3.87°C.
+
+[atmosphere]: https://access-hive.github.io/website/pr-preview/pr-157/model_components/atmosphere/
```
Comment (rbeucher) [inline docs/configurations/access-esm.md:17]: Why do links point to the Preview?

File: docs/configurations/access-om.md [hunk 3]
```diff
@@ -3,19 +3,54 @@
...
+
+ACCESS-OM2 is a suite of coupled ocean-sea ice models developed by the [Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA)][COSIMA]. All models use the [MOM5.1][MOM5.1] ocean model coupled to the [CICE5.1.2][CICE5.1.2] sea ice model via OASIS3-MCT.
 
 The models in the ACCESS-OM2 differ by their grid spatial resolution:
 
  - [ACCESS-OM2][ACCESS-OM2] at 1° with 50 vertical levels
  - [ACCESS-OM2-025][ACCESS-OM2-025] at 0.25° with 50 vertical levels
  - [ACCESS-OM2-01][ACCESS-OM2-01] at 0.1° with 75 vertical levels
 
+## [JRA55-do]  {{ recom }}
```
Comment (rbeucher) [inline docs/configurations/access-om.md:20]: Should that also be added to datasets in the MED section?
Same for ERA5

**Review by rbeucher — COMMENTED** [review]
Text is good. Some practical questions about link and wheter the information should also be copied in the MED Datasets section.

File: docs/configurations/access-cm.md (hunk 1 above)
Comment (aidanheerdegen) [reply docs/configurations/access-cm.md:15]: I have slight misgivings about linking specific versions of models to generic pages about those models. There are links to components in the blurb above. Is that insufficient?

File: docs/configurations/access-esm.md (hunk 2 above)
Comment (aidanheerdegen) [reply docs/configurations/access-esm.md:17]: Mistake. Thanks for picking it up.

File: docs/configurations/access-om.md (hunk 3 above)
Comment (aidanheerdegen) [reply docs/configurations/access-om.md:20]: Yes. Will do so.

Comment (aidanheerdegen) [reply docs/configurations/access-om.md:20]: Yep. Will do so. Tah.

File: docs/configurations/access-cm.md (hunk 1 above)
Comment (rbeucher) [reply docs/configurations/access-cm.md:15]: OK

File: .github/workflows/mlc_config.json [hunk 4]
```diff
@@ -9,6 +9,9 @@
         {
             "pattern": "linkedin.com"
         },
+        {
+            "pattern": "https://opus.nci.org.au/display/Help/ARE+User+Guide"
```
Comment (rbeucher) [inline .github/workflows/mlc_config.json:13]: Why do we want to ignore it. It's just down today right?

Comment (aidanheerdegen) [reply .github/workflows/mlc_config.json:13]: Nope, isn't down. I've checked it twice and both times it was fine. For whatever reason GitHub actions can't see it.

I *love* the link checker BTW. So good! I want to add a scheduled run so we don't only check for PRs, but you know, time etc.

**Comment by aidanheerdegen** [comment]
Sorry for spamming this with broken link checks @rbeucher I was genuinely curious what would work, so have resisted the "correct" formulation you've used in other pages on purpose.

**Comment by aidanheerdegen** [comment]
Don't merge yet, have a couple of links to fix

**Comment by aidanheerdegen** [comment]
@rbeucher ready for re-review

File: README.md [hunk 5]
```diff
@@ -1,9 +1,13 @@
...
+[![ACCESS-Hive Badge](docs/assets/badge.svg)](https://access-hive.github.io/website/)
+
+Documentation hub for the Earth System models, ACCESS, and their community.
+
+This repository uses [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) to build the website which accessible at this URL:
+
+https://access-hive.github.io/website/
 
-Documentation hub for the Earth System models, ACCESS, and their community
+If you wish to add documentation to the ACCESS-Hive website see the [contribution guide](https://access-hive.github.io/website/pr-preview/pr-157/about/contribute/) for instructions.
```
Comment (rbeucher) [inline README.md:11]: We still have a link to the PR here

File: docs/configurations/access-esm.md [hunk 6]
```diff
@@ -1,3 +1,27 @@
...
+
+[**Citation**][ACCESS-ESM1.5-cite] |
+[**Tutorial**][ACCESS-ESM1.5-tute]
+
+ACCESS-ESM1.5 is a fully-coupled climate model with land and ocean carbon cycle components. ACCESS-ESM1.5 has mainly been developed to enable Australia to participate in the Coupled Model Intercomparison Project Phase 6 (CMIP6) with an ESM version. An assessment of the climate response to CO2 forcing indicates that ACCESS-ESM1.5 has an equilibrium climate sensitivity of 3.87°C.
+
+[atmosphere]: ../model_components/atmosphere.md
+[land]: ../model_components/land.md
+[land biogeochemistry]: ../model_components/bgc_land.md
+[ocean]: ../model_components/ocean.md
```
Comment (rbeucher) [inline docs/configurations/access-esm.md:20]: so #model_components/ocean.md would not work?

**Review by rbeucher — COMMENTED** [review]
One more link to fix ;-)

File: README.md (hunk 5 above)
Comment (aidanheerdegen) [reply README.md:11]: Thanks for spotting. Clearly I need to look when I copy and paste ... :|

File: docs/configurations/access-esm.md (hunk 6 above)
Comment (aidanheerdegen) [reply docs/configurations/access-esm.md:20]: Didn't try that one. I looked and saw you'd used `../` style relative links. So `#` anchors the link to the base of `docs`, e.g. the site?

Comment (rbeucher) [reply docs/configurations/access-esm.md:20]: I'm wondering. Which one is correct?

Comment (aidanheerdegen) [reply docs/configurations/access-esm.md:20]: Testing now.

File: README.md [hunk 7]
```diff
@@ -1,9 +1,13 @@
...
+[![ACCESS-Hive Badge](docs/assets/badge.svg)](https://access-hive.github.io/website/)
+
+Documentation hub for the Earth System models, ACCESS, and their community.
+
+This repository uses [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) to build the website which accessible at this URL:
+
+https://access-hive.github.io/website/
 
-Documentation hub for the Earth System models, ACCESS, and their community
+If you wish to add documentation to the ACCESS-Hive website see the [contribution guide](https://access-hive.github.io/website/about/contribute/) for instructions.
```
Comment (rbeucher) [inline README.md:10]: I'm going to be annoying... Why not use an anchor link?

File: docs/configurations/access-esm.md (hunk 6 above)
Comment (rbeucher) [reply docs/configurations/access-esm.md:20]: Ok does not work then

Comment (aidanheerdegen) [reply docs/configurations/access-esm.md:20]: That didn't work, have reverted. Perhaps I didn't it incorrectly. I'll leave as-is as it is working.

File: README.md (hunk 7 above)
Comment (aidanheerdegen) [reply README.md:10]: No problem. but I don't understand. An anchor into the contribute page somewhere?

Comment (rbeucher) [reply README.md:10]: I think this is OK.

**Review by rbeucher — APPROVED** [review]

**Comment by rbeucher** [comment]
You can merge.

**Comment by aidanheerdegen** [comment]
Will do.

**Comment by aidanheerdegen** [comment]
Did a squash as that was one hell-untidy set of commits. Next time I'll try and remember to do a nicer rebase and rewrite the history a bit before final sign-off.

**Comment by ccarouge** [comment]
@aidanheerdegen so far we have done squashes pretty exclusively. Do we want to have a set method or the person who merges chooses?

**Comment by aidanheerdegen** [comment]
I thought we'd decided not to squash? Maybe I am remembering a different conversation. I'm a big squasher, probably because I'm such an untidy committer! :)

The only downside to squash is losing some atomicity of commits. If the commit history is clean in the PR I don't have a problem with a straight merge.

**Pull request merged**